plenum/test/node_catchup/test_remove_request_keys_post_catchup.py

```diff
@@ -56,13 +57,10 @@ def chk(key, nodes, present):
     # Reset catchup reply delay so that  catchup can complete
     slow_node.nodeIbStasher.reset_delays_and_process_delayeds(CatchupRep.typename)
 
-    old_last_ordered = fast_nodes[0].master_replica.last_ordered_3pc
-
     ensure_view_change(looper, txnPoolNodeSet)
     ensureElectionsDone(looper, txnPoolNodeSet)
 
-    ensure_all_nodes_have_same_data(looper, fast_nodes)
-    assert slow_node.master_replica.last_ordered_3pc == old_last_ordered
+    ensure_all_nodes_have_same_data(looper, txnPoolNodeSet)
```

Comment:
Was switch from `fast_nodes` to full `txnPoolNodeSet` intentional?

Reply:
because for now all of the delayed requests will be reordered and for slow node too